static/templates/actors/npc/partials/attack.hbs

```diff
@@ -57,4 +43,10 @@
             </div>
         {{/if}}
     </div>
+
+    {{#if action.description}}
+        <div class="description">
```

Comment:
```suggestion
        <div class="description item-summary">
```

Reply:
Unfortunately that causes the save and restore state to render it hidden.